config.mk

```diff
@@ -1,6 +1,6 @@
 ##### Options which a user might set before building go here #####
 
-# Where to install idris2 binaries and libraries
+# Where to install idris2 binaries and libraries (must not be a relative path)
```

Comment:
Potentially change to "(must be an absolute path)"? I can't really decide if it's clearer or not  ^^;

Reply:
I suppose the positive view would be more constructive than a [Not](https://github.com/idris-lang/Idris2/blob/master/libs/prelude/Prelude/Basics.idr#L11)!

Reply:
Corrected: 722bab81100e3272971cc5f0141e8a2ef8aa451d